q2_cutadapt/tests/test_demux.py

```diff
@@ -501,7 +501,7 @@ def test_di_mismatched_barcodes(self):
                                  forward_barcodes=forward_barcodes,
                                  reverse_barcodes=reverse_barcodes)
 
-    def test_di_duplicate_barcode_pairs(self):
+    def test_di_duplicate_barcode_pairs_not_mixed(self):
```

Comment:
```suggestion
    def test_dual_index_duplicate_barcode_pairs_not_mixed(self):
```
This will be easier to understand for anyone who is just looking through the code in the future.